experimental/cloudstrapper/README.md

```diff
@@ -0,0 +1,178 @@
+      - buildAwsAz indicates an AZ within the region specified above
+
+  - build-provision: Setup build instance using default security group, Bootkey and Ubuntu with
+    t2.xlarge. Optionally, Provision a AGW compliant image (Debian 4909 or Ubuntu 20.04) 
+    ```
+    ansible-playbook build-provision.yaml
+    ```
+
+  - build-configure: Configure build instance by setting up necessary parameters and reading from
+    dynamic inventory. The build node was provisioned with the tag Name:buildOrc8r in this example.
+    ```
+    ansible-playbook build-configure.yaml -i ~/magma-experimental/files/common_instance_aws_ec2.yaml -e "buildnode=tag_Name_buildOrc8r" -e "ansible_python_interpreter=/usr/bin/python3"
+    ```
+
+  - build-ami-configure: Configure AMI for AGW by configuring base AMI image with AGW packages and
+    building OVS.
+    *TODO: Remove existing snapshot and create new snapshot*
+    ```
+    ansible-playbook build-ami-configure.yaml -e '@vars/main.yaml' -i files/build_instance_aws_ec2.yaml -e "buildnode=tag_Name_ec2MagmaBuild" -u admin
+    ```
+
+  - Result: Build instance created, images and helm charts published. AGW AMI created.
+
+## 4. Control Plane/Cloud Services
+
+  The control- roles deploy and configure orc8r in a target region.
+
+  - Create control plane elements: Provision, Configure and Init
+    Observe the variables set in cluster.yaml
+
+    Make any custom changes to main.tf here before initializing. If you would like to persist changes
+    across re-installs, make changes to the main.tf.j2 Jinja2 template file directly so that the custom
+    configuration be used across every terraform init.
+
+  - Requires: secrets.yaml in the dirInventory folder. Use the sample file in roles/vars/secrets.yaml
+  - Orchestrator : Deploy orchestrator 
+  ```
+    ansible-playbook orc8r.yaml [ --skip-tags deploy-orc8r ]
+  ```
+
+  Note: First time installs might want to skip using Terraform from within Ansible to make sure the
+  new build works as expected. When using a stable build, the tag does not have to be skipped.
+
+  - Result: Orchestrator certificates created, Terraform files initialized, Orchestrator deployed
+    via Terraform
+
+## Known Issues, Best Practices & Expected Behavior
+
+### Best Practices
+    
+    1. Although the deployment will work from any Ubuntu host, using the Cloudstrapper AMI might be
+       the quickest way to get the deployment going since it includes all the necessary dependencies
+       in-built. 
+
+    2. The tool is customizable to build every desired type of installation. However, for initial
+       efforts, it might be better to to use the existing default values. 
+
+### Expected Behavior - Install
+
+    1. Prior code base
+       If a prior code base resides in the home folder, install exists reporting code already exists.
+       This is done to ensure the user is aware that a prior code base exists and needs to be moved
```

Comment:
Add any commands user will have to do to move the code and add or pull new code.

Reply:
Adding documentation to indicate code directory can be renamed.